stable/verdaccio/templates/ingress.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.ingress.enabled }}
+{{- $serviceName := include "verdaccio.fullname" . -}}
+{{- $servicePort := .Values.service.port -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "verdaccio.fullname" . }}
+  labels:
+    app: {{ template "verdaccio.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    {{- range $key, $value := .Values.ingress.annotations }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  rules:
+    {{- range $host := .Values.ingress.hosts }}
+    - host: {{ $host }}
+      http:
+        paths:
+          - path: /
```

Comment:
@chainlink: path: `/` does not automatically add a wildcard match in GCE so this ingress will only exactly match `/`. Removing the path entirely help in my specific case but atleast I would need `/*` to serve anything but the root html.

Reply:
Should we default to` /*` then?

Reply:
I would certainly like that. I hope and think that Nginx-ingresses wont be affected in a negative way.

Reply:
Just hit this issue as well, using `/` will only route the root and not all other child routes need to serve the UI and more. Changing it to `/*` will work.